ACKategoriesCore/PropertyWrappers/UserDefault.swift

```diff
@@ -64,12 +65,36 @@ public struct UserDefault<Value: Codable> {
                     errorLogger?(error)
                 }
             }
+
+            if #available(iOS 13.0, macOS 10.15, *) {
+                subject().send(newValue)
+            }
+        }
+    }
+
+    @available(iOS 13.0, macOS 10.15, *)
+    public var projectedValue: CurrentValueSubject<Value, Never> {
+        subject()
+    }
+
+    @available(iOS 13.0, macOS 10.15, *)
+    private func subject() -> CurrentValueSubject<Value, Never> {
+        if let subject = objc_getAssociatedObject(self, &AssociationKeys.subject) as? CurrentValueSubject<Value, Never> {
```

Comment:
❓ why do we need an associated object in this case? Wouldn't be a simple property to store the subject enough? I thought that we need it when the property wrapper is `struct`

Reply:
You cannot make stored property that has limited availability.